test/unittests/execute_test.cpp

```diff
@@ -380,9 +380,9 @@ TEST(execute, i32_load_all_variants)
         from_hex("0061736d0100000001060160017f017f030201000504010101010a0901070020002802000b");
     const auto module = parse(wasm);
 
-    auto& load_instr = const_cast<uint8_t&>(module->codesec[0].instructions[1]);
-    ASSERT_EQ(load_instr, Instr::i32_load);
-    ASSERT_EQ(module->codesec[0].immediates.substr(4), "00000000"_bytes);  // load offset.
+    auto* const load_instr = const_cast<uint8_t*>(&module->codesec[0].instructions[1]);
+    ASSERT_EQ(*load_instr, Instr::i32_load);
+    ASSERT_EQ(bytes_view(load_instr + 1, 4), "00000000"_bytes);  // load offset.
```

Comment:
Is this `bytes_view` alternative better because it is faster, or does it result in a nicer error display?

Reply:
I think it was easier for me to read it like that when modifying the test.